server/routes/service-provider/action-plan/add-activities/addActionPlanActivitiesView.ts

```diff
@@ -1,16 +1,31 @@
 import ViewUtils from '../../../../utils/viewUtils'
 import AddActionPlanActivitiesPresenter from './addActionPlanActivitiesPresenter'
+import { BackLinkArgs } from '../../../../utils/govukFrontendTypes'
 
 export default class AddActionPlanActivitiesView {
   constructor(private readonly presenter: AddActionPlanActivitiesPresenter) {}
 
+  private get backLinkArgs(): BackLinkArgs | null {
+    if (this.presenter.activityNumber === 1) {
+      return null
```

Comment:
We can perhaps this returning back to the progress page rather than null?

Reply:
yeh i did think about that. i think we don't want to encourage users to exit this journey once started though. we can ask the UX people tomorrow.